packages/databyss-ui/primitives/Control/native/TextInput.js

```diff
@@ -31,6 +32,7 @@ const TextInput = forwardRef(
         onChange={e => onChange({ textValue: e.target.value })}
         ref={ref}
         css={[resetCss].concat(concatCss)}
+        autoFocus={autoFocus}
```

Comment:
What does this prop do?

Reply:
It autofocuses an input: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefautofocus. I used it for when we switch route to /search so it autofocus the search input and the user can start typing right away